oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexNameHelper.java

```diff
@@ -39,6 +40,26 @@ public static String getIndexAlias(String indexPrefix, String indexPath) {
         return getElasticSafeIndexName(indexPrefix + "." + indexPath);
     }
 
+    /**
+     * Create a name for remote elastic index from given index definition and seed.
+     * @param indexDefinition elastic index definition to use
+     * @param seed seed to use
+     * @return remote elastic index name
+     */
+    public static String getRemoteIndexName(ElasticIndexDefinition indexDefinition, long seed) {
+        return getElasticSafeIndexName(
```

Comment:
I can even make this method `getElasticSafeIndexName` private. Other than this class, it is being used in benchmark tests - https://github.com/oak-indexing/jackrabbit-oak/blob/46d1f1c3f9ff193c308f7b15a45dd0d7dcd75bd8/oak-benchmarks-elastic/src/main/java/org/apache/jackrabbit/oak/benchmark/util/TestHelper.java#L52

That logic won't work anyway because we won't have reindexCount as suffix. I think alias can directly be used for deletion there.

Reply:
Made the method private and fixed benchmark tests.